src/classes/HH_Container_LCTRL.cls

```diff
@@ -37,6 +37,28 @@
 public class HH_Container_LCTRL {
     
     /*******************************************************************************************************
+    * @description trick the packaging spider to pickup references to custom labels only used by our
+    * lightning components.
+    */
+    public static void packageLabels() {
```

Comment:
https://github.com/SalesforceFoundation/Cumulus/pull/2078/files/afb6a9f7ab92cbcce58c8259500251e8f36e8dda#diff-06ad8a1650536fafd5d1c9efc19c965bR105 refers to `$Label.c.Street` and `$Label.npsp.Street`.  Other labels in this section refer to similarly named labels.  Maybe this were supposed to be `$Label.npsp.lblStreet`, etc.?